portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,126 @@
+          <span class="m-left name">e</span>
+        </h1>
+        <p>Interning at Google in the summer of 2020 <br> Part of Student Training in Engineering Program</p>
+        <a href="#about" onclick="addRandomGreeting()">begin</a>
+      </div>
+      <div class="avatar">
+        <img src="./image/avatar.JPG" alt="Lee's profile picture">
+      </div>
+    </div>
+
+    <div class="space" id="about"></div>
+
+    <div class="container about">
+      <h1>About Me</h1>
+      <div class="about-me-info">
+        <p>I am a rising sophomore at Yale University.  I am interested in UX and front-end development.
+        My goal is to make accessible and intuitive technology for everyone but especially for elders.
+        I am majoring in computer science while also taking classes in psychology and visual arts. In my free time, I enjoy making music,
+      creating art, baking pastries, and trying new things. </p>
+      </div>
+        <img src="./image/red.jpg" alt="">
+        <div class="secret-message"><div id="message-container"></div></div>
+      </div>
+    </div>
+
+    <div class="space" id="skills"></div>
+    <div class="container skills">
+      <h1>Skills</h1>
+      <p class="skills-info"> In past projects, I have mainly worked on designating tasks and handling the front-end components.
+        I love art, so I enjoy using HTML/CSS and frameworks like React and Bootstrap to create aesthetic and straightforward platforms.
+        In my free time, I am also taking a self-paced full stack web developer course to enhance my understanding of Python and Django, JavaScript, and jQuery. </p>
+      <div class="html">
+        <div class="tag-html">
+          <p>HTML/CSS</p>
+        </div>
+        <div class="progressbar">
+          <div class="bar-html" id="htmlBar"></div>
+        </div>
+      </div>
+      <div class="html">
+        <div class="tag-javascript">
+          <p>JavaScipt</p>
+        </div>
+        <div class="progressbar">
+          <div class="bar-javascript" id="jsBar"></div>
+        </div>
+      </div>
+      <div class="html">
+        <div class="tag-python">
+          <p>Python</p>
+        </div>
+        <div class="progressbar">
+          <div class="bar-python" id="pythonBar"></div>
+        </div>
+      </div>
+      <div class="html">
+        <div class="tag-java">
+          <p>Java</p>
+        </div>
+        <div class="progressbar" >
+          <div class="bar-java" id="javaBar"></div>
+        </div>
+      </div>
+    </div>
+
+    <div class="space" id="contact"></div>
+
+    <div class="container contact">
+      <h1>Reach me</h1>
+      <div class="space" id="contact"></div>
+      <div class="contact-me ">
+        <!-- Not sure why the icons links move down with a:hover -->
+        <a href="https://github.com/leeannachen"><i class="fab fa-github-square fa-5x"></i></a>
+        <a href="https://www.linkedin.com/in/unavailable/"> <i class="fab fa-linkedin fa-5x"></i> </a>
```

Comment:
I encourage you to make a linked in page if you don't already have one. Recruiters actually look at them. That is how I ended up at google even though my linked in page is pretty plain.